extension/src/plots/webview/messages.ts

```diff
@@ -101,9 +103,15 @@ export class WebviewMessages {
       case MessageFromWebviewType.SELECT_EXPERIMENTS:
         return this.selectExperimentsFromWebview()
       case MessageFromWebviewType.REMOVE_CUSTOM_PLOTS:
-        return commands.executeCommand(RegisteredCommands.PLOTS_CUSTOM_REMOVE)
+        return commands.executeCommand(
+          RegisteredCommands.PLOTS_CUSTOM_REMOVE,
+          this.dvcRoot
+        )
       case MessageFromWebviewType.REFRESH_REVISIONS:
-        return this.refreshData()
+        return commands.executeCommand(
+          RegisteredCommands.PLOTS_REFRESH,
```

Comment:
[F] We had code and an extra command in here that was really duplicating logic unnecessarily.